.github/workflows/unit_test.yml

```diff
@@ -27,18 +27,27 @@ jobs:
       env:
           GOPATH: /home/runner/go
           GOBIN: /home/runner/go/bin
-    - name: install bcc
+    - name: install libbpf
       uses: sustainable-computing-io/kepler-action@v0.0.3
       with:
-        ebpfprovider: bcc
+        ebpfprovider: libbpf
     - name: Prepare environment 
       run: |
           sudo apt-get install -y cpuid
           cd doc/ && sudo ./dev/prepare_dev_env.sh && cd - 
           git config --global --add safe.directory /kepler
     - name: Run 
       run: |
-          make test-verbose
+          sudo apt remove libbpf-dev
```

Comment:
if `libbpf-dev` is not enough of our libbpf driver, we should make a deb package at https://github.com/sustainable-computing-io/kepler-ci-artifacts and update our kepler-action.

Reply:
https://github.com/sustainable-computing-io/kepler-action/issues/87 created accordingly.

Reply:
@rootfs / @vimalk78 / @sthaha could you please help double confirm if we need sustainable-computing-io/kepler-action#87 ?

Reply:
new issue related to this PR as https://github.com/sustainable-computing-io/kepler/issues/1086

Reply:
> if `libbpf-dev` is not enough of our libbpf driver, we should make a deb package at 

yes @SamYuan1990, opening a PR for the same. in this PR i wanted to test what works. will update kepler-action, and then we dont needs this setup in CI

Reply:
@SamYuan1990 pls check this PR https://github.com/sustainable-computing-io/kepler-action/pull/90 
Cc: @rootfs